app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ActiveRecord::Base
   has_secure_password
 
+  has_many :user_group_memberships
+  has_many :groups, through: :user_group_memberships
```

Comment:
@Enriikke I like the `has_many :through` relationship here but what do you think? Would you have done it this way? Also, I've called our third model `UserGroupMemberships` as opposed to something like `Memberships` just to be really explicit that this is a join table for those two. Also, would you call this a join table :confused:? I didn't have to create one manually like you do with `has_and_belongs_to_many ` but that's still the correct term for `user_group_memberships` perhaps?